src/service.js

```diff
@@ -323,11 +323,12 @@ export function getTopWindow(win) {
 /**
  * Returns the parent "friendly" iframe if the node belongs to a child window.
  * @param {!Node} node
- * @param {!Window} topWin
+ * @param {!Window=} opt_topWin
  * @return {?HTMLIFrameElement}
  */
-export function getParentWindowFrameElement(node, topWin) {
+export function getParentWindowFrameElement(node, opt_topWin) {
```

Comment:
seems to me quite random to have this method here. It's only used by amp-analytics.
We can either move it to amp-analytics, or move it to friendly-iframe-embed.js for future reuse.

Reply:
Hmm. It's used in many places.

Reply:
You're right. Still be good to move out to friendly-iframe-embed.js. but can be a different PR